src/utils/courseSequenceMapping.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Would be great to have someone from algo1 review this :)

Reply:
@SENG499-S22-Company3/algorithm-1 Would you mind taking a look at this mapping when you guys have a chance?

Reply:
Seems good for the most part, but I'm unsure if this supports Rich's input for the number of sections to create for a given course. Since this needs to be passed to alg-1. If it's not passed to algorithm-1, then company 3 and 4 alg-1 would split up a course with an oversized capacity even though Rich left the course to be extra large on purpose.

Reply:
I don't believe that numSections needs to be stored in the database as it is going directly from the frontend 'generate schedule' mutation to algorithm 1, so it should be able to reside in memory. I'll verify with the rest of backend tomorrow.

Reply:
If that's the case it should be fine then.